app/models/UserAnswers.scala

```diff
@@ -50,6 +50,9 @@ final case class UserAnswers(
     }
   }
 
+  def unsafeSet[A](page: Settable[A], value: A)(implicit writes: Writes[A]): UserAnswers =
+    set(page, value).get
```

Comment:
Not sure about defining unsafe methods outside of test scenarios

Reply:
Yeah me neither, I I'll go back and add it as an extension method